### PR TITLE
Adjusting the docstrings for a missing exception. 

### PR DESCRIPTION
**Pull request by timsloan**
Cleaning up some code for readability. Adding PMMailInactiveRecipient Exception to the doc string. No functionality changes.

### CODE REVIEW

File: postmark/django_backend.py [hunk 1]
```diff
@@ -135,7 +137,7 @@ def _send(self, messages):
                 return False
         else:
             pm_messages = list(map(self._build_message, messages))
-            pm_messages = [m for m in pm_messages if m != False]
+            pm_messages = [m for m in pm_messages if not m]
```
Comment (nicholasserra) [inline postmark/django_backend.py]: This looks like an incorrect substitution, shouldn't it be `if m`? Original condition is `if m is not false` where the change is now `if m is falsey`

Comment (timsloan) [reply postmark/django_backend.py]: Oops. Good catch. @nicholasserra

Comment (nicholasserra) [reply postmark/django_backend.py]: Looks like a spot where I need to add some test coverage :P

**Comment by nicholasserra** [comment]
Thanks!

**Pull request merged**